templates/case/information.html.twig

```diff
@@ -31,7 +31,7 @@
         <div class="col-auto">
             {# TODO: Needs functionality on buttons #}
             <a name="" id="" class="btn btn-success" href="#" role="button">{% trans %}Create representative{% endtrans %}</a>
-            <a name="" id="" class="btn btn-primary" href="#" role="button">{% trans %}Edit basic information{% endtrans %}</a>
+            <a name="" id="" class="btn btn-primary" href="{{ url('case_information_edit', {id: case.id}) }}" role="button">{% trans %}Edit basic information{% endtrans %}</a>
```

Comment:
The value for the route name parameter in the url function should be `case_edit`.

Reply:
Used renamed route name `case_edit.`